### PR TITLE
Added default cookies options support

### PR DESCRIPTION
**Pull request by AlfonsoUceda**
This allow to send default cookies values to every cookie.

Values set for a cookie by user are respected.

### CODE REVIEW

File: lib/lotus/action/cookie_jar.rb [hunk 1]
```diff
@@ -77,6 +78,21 @@ def []=(key, value)
...
 
       private
+
+      # Merge default cookies options with values provided by user
+      #
+      # Cookies values provided by user are respected
+      #
+      # @since x.x.x
+      # @api private
+      def _merge_default_values(value)
```
Comment (AlfonsoUceda) [inline lib/lotus/action/cookie_jar.rb]: I don't like a lot this method. any idea?

File: lib/lotus/controller/configuration.rb [hunk 2]
```diff
@@ -558,14 +592,15 @@ def duplicate
...
-        @default_headers    = {}
-        @action_module      = ::Lotus::Action
+        @handle_exceptions       = true
+        @handled_exceptions      = {}
+        @modules                 = []
+        @formats                 = DEFAULT_FORMATS.dup
+        @default_format          = nil
+        @default_charset         = nil
+        @default_headers         = {}
+        @default_cookies_options = {}
```
Comment (jodosha) [inline lib/lotus/controller/configuration.rb]: This should be `{httponly: true}`, so we enforce a secure policy by default.

File: lib/lotus/action/cookie_jar.rb (hunk 1 above)
Comment (jodosha) [inline lib/lotus/action/cookie_jar.rb]: What's the thing that you don't like?

File: lib/lotus/controller/configuration.rb (hunk 2 above)
Comment (AlfonsoUceda) [inline lib/lotus/controller/configuration.rb]: do you think this should be here instead lotus gem?

File: lib/lotus/action/cookie_jar.rb (hunk 1 above)
Comment (AlfonsoUceda) [inline lib/lotus/action/cookie_jar.rb]: the use of is_a? method, but I think I dont have more options

File: lib/lotus/controller/configuration.rb (hunk 2 above)
Comment (jodosha) [inline lib/lotus/controller/configuration.rb]: Oh, you're right. Sorry about the confusion. This is good :+1:

File: lib/lotus/controller/configuration.rb [hunk 3]
```diff
@@ -498,6 +498,39 @@ def default_headers(headers = nil)
...
+      #       domain: 'lotusrb.org',
+      #       path: '/controller',
+      #       secure: true,
+      #       httponly: true
+      #     })
+      #   end
+      def default_cookies_options(options = nil)
+        if options
+          @default_cookies_options.merge!(
+            options.reject {|_,v| v.nil? }
```
Comment (joneslee85) [inline lib/lotus/controller/configuration.rb]: nitpick, please add space after `{`

**Comment by joneslee85** [comment]
:+1:

**Pull request merged**